### ISSUE TITLE
unable to connect to dbus

### ISSUE THREAD

**Issue by DrOllyGomez** [LooseLab/bulkvis#32]
Hiya. Keen on using bulkvis.
Managed to get some of the way through the install.

Q1: does a 'bulkfile' = a fast5 file that contains other, individual fast5 files?
Q2: did basecalling need to be enabled? Can I just look at squiggles?

Here's a screenshot of something happening..... 

![Screenshot at 2020-01-16 12-15-31](https://user-images.githubusercontent.com/50023481/72524521-00366b80-385a-11ea-9f4d-40dafdd10d4a.png)

... but logging to console says the following.....

~/.virtualenvs/bulkvis/bulkvis$ bokeh serve --show ~/.virtualenvs/bulkvis/bulkvis
2020-01-16 12:12:17,847 Starting Bokeh server version 0.12.15 (running on Tornado 5.0.1)
2020-01-16 12:12:17,849 Bokeh app running at: http://localhost:5006/bulkvis
2020-01-16 12:12:17,849 Starting Bokeh server with process id: 6963

** (/usr/lib/firefox/firefox:7105): WARNING **: 12:12:18.435: Unable to connect to dbus: Could not connect: Connection refused
[2020-01-16T12:12:18Z ERROR audio_thread_priority::rt_linux] setrlimit64: 1
/home/mike.harbour/.virtualenvs/bulkvis/lib/python3.6/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters

** (/usr/lib/firefox/firefox:7150): WARNING **: 12:12:18.903: Unable to connect to dbus: Could not connect: Connection refused
2020-01-16 12:12:18,908 200 GET /bulkvis (127.0.0.1) 177.64ms
2020-01-16 12:12:19,112 200 GET /static/css/bokeh.min.css?v=f7fe43f4980a82921415acbd60d5fd66 (127.0.0.1) 7.16ms
2020-01-16 12:12:19,316 101 GET /bulkvis/ws?bokeh-protocol-version=1.0&bokeh-session-id=qV2dJR0Tsr3asgoqy4jSoZ3X7jM3tLA69dQJIefqqZVN (127.0.0.1) 1.10ms
2020-01-16 12:12:19,316 WebSocket connection opened
2020-01-16 12:12:19,316 ServerConnection created

** (/usr/lib/firefox/firefox:7213): WARNING **: 12:12:19.502: Unable to connect to dbus: Could not connect: Connection refused

....symptom is, no bulkfiles seem to be available to choose.
I put a multi-fast5 file into the folder stated to be the 'bulkfile directory' in my config.ini.

Any help gratefull received
Mike

**Comment by epi-gene** [comment]
I still have the same issue. Were you able to fix it ?

**Comment by DrOllyGomez** [comment]
Unfortunately not.

**Comment by mattloose** [comment]
Ah. I’m sorry about this - we will look in to it for you ASAP.

**Comment by epi-gene** [comment]
Ok. Reinstallation has resolved the dbus error - don't know how. But I still cannot see any fast5 files to select.

**Comment by DrOllyGomez** [comment]
are you saying that you are able to replicate the behaviour I've seen?

**Comment by epi-gene** [comment]
Yes. I am able to go through the installation step and start bokeh. Firefox opens bulkvis, but I still cannot select any fast5 files that are in the folder that I specified in cofig.ini

My terminal shows the following

2020-03-12 10:42:24,140 Starting Bokeh server version 0.12.15 (running on Tornado 5.0.1)
2020-03-12 10:42:24,142 Bokeh app running at: http://localhost:5006/bulkvis
2020-03-12 10:42:24,142 Starting Bokeh server with process id: 7271
/home/epigene/bulkvis-env/lib/python3.6/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2020-03-12 10:42:24,380 200 GET /bulkvis (127.0.0.1) 133.95ms
2020-03-12 10:42:24,419 200 GET /bulkvis (127.0.0.1) 21.60ms
2020-03-12 10:42:24,526 101 GET /bulkvis/ws?bokeh-protocol-version=1.0&bokeh-session-id=dYTWP7LTYlcRvapEOjhGUDEJdSoU3Lf9sc0uNQwg6CL7 (127.0.0.1) 0.37ms
2020-03-12 10:42:24,526 WebSocket connection opened
2020-03-12 10:42:24,526 ServerConnection created

**Comment by DrOllyGomez** [comment]
Ok, I was worried I was wasting your time by having done something incorrectly.

**Comment by epi-gene** [comment]
@DrOllyGomez If you are trying to visualize squiggles, you can either use SquiggleKit - https://github.com/Psy-Fer/SquiggleKit or hdf view - https://www.hdfgroup.org/downloads/hdfview/ both of these are working pretty well for me. You don't need basecalled fast5's for theses

**Comment by DrOllyGomez** [comment]
many thanks for the references.
